docker/src/python/assign-class-label/mutate.py

```diff
@@ -0,0 +1,106 @@
+import logging
+import json
+import base64
+import os
+from flask import Flask, request, jsonify
+from kubernetes import config, client
+from openshift.dynamic import DynamicClient
+
+webhook = Flask(__name__)
+
+LOG = logging.getLogger(__name__)
+logging.basicConfig(level=logging.INFO)
+
+try:
+    config.load_incluster_config()
+except config.ConfigException as e:
+    LOG.error("Could not configure Kubernetes client: %s", str(e))
+    exit(1)
```

Comment:
To make this easier to run locally, rewrite this to use `config.load_kube_config()` if `config.load_incluster_config()` fails. Something like:


```suggestion
try:
    config.load_incluster_config()
except config.ConfigException:
    try:
        config.load_kube_config()
    except config.ConfigException:
        LOG.error("Could not configure Kubernetes client")
        exit(1)
```